### ISSUE TITLE
Bug 

### ISSUE THREAD

**Issue by RodyaUshakov** [Kxnrl/MiniGames#10]
When issuing a knife at the beginning, the damage goes.If you do not dispense a knife at the beginning, it will not dispense a knife in modes

**Comment by RodyaUshakov** [comment]
and the data that I enter into the cfg does not respond, for example, to remove the left in the skillboard tab

**Comment by RodyaUshakov** [comment]
[MG] Rodya pressed 348<51246>(spawn_b6) and can I remove it?

**Comment by Kxnrl** [comment]
> [MG] Rodya pressed 348<51246>(spawn_b6) and can I remove it?

yes, we have a convar ``mg_button_watcher``

**Comment by RodyaUshakov** [comment]
> > [MG] Rodya pressed 348<51246>(spawn_b6) and can I remove it?
> 
> yes, we have a convar `mg_button_watcher`

and the data that I enter into the cfg does not respond, for example, to remove the left in the skillboard tab?
When issuing a knife at the beginning, the damage goes.If you do not dispense a knife at the beginning, it will not dispense a knife in modes

**Comment by Kxnrl** [comment]
you need to install ammomanager plugin

**Comment by RodyaUshakov** [comment]
> you need to install ammomanager plugin

it is installed, without a single error

**Comment by Kxnrl** [comment]
so game_player_equip still not give knife to players?

**Comment by RodyaUshakov** [comment]
> so game_player_equip still not give knife to players?

no, but if you give out a knife at the beginning of the round, then it deals damage

**Comment by Kxnrl** [comment]
> > so game_player_equip still not give knife to players?
> 
> no, but if you give out a knife at the beginning of the round, then it deals damage

ofc, so you need make map configs for each map

**Comment by RodyaUshakov** [comment]
> > > so game_player_equip still not give knife to players?
> > 
> > 
> > no, but if you give out a knife at the beginning of the round, then it deals damage
> 
> ofc, so you need make map configs for each map

I have them made, if I turn on the knife at the beginning of the glad, then it does damage. If I turn it off at the beginning of the round, the knife is not issued in mini games

**Comment by Kxnrl** [comment]
never seen that problem, what map?
the shared configs can be found here https://github.com/fys-csgo/servers-config
license under CC4.0 NC SA

**Comment by RodyaUshakov** [comment]
> never seen that problem, what map?
> the shared configs can be found here https://github.com/fys-csgo/servers-config
> license under CC4.0 NC SA

Ok, thanks, I'll try and write, unfortunately my card is not on your list

**Comment by RodyaUshakov** [comment]
> never seen that problem, what map?
> the shared configs can be found here https://github.com/fys-csgo/servers-config
> license under CC4.0 NC SA

On some maps, a knife still drops in knife mode and cannot be picked up

**Comment by Kxnrl** [comment]
map name?

**Comment by RodyaUshakov** [comment]
> map name?

for some reason it became on all maps

**Comment by Kxnrl** [comment]
> > map name?
> 
> for some reason it became on all maps

sorry, I can't help. It works on my server.  I don't know what's wrong.

my server here: https://fyscs.com
